libraries/AP_ESC_Telem/AP_ESC_Telem.cpp

```diff
@@ -105,6 +105,22 @@ bool AP_ESC_Telem::get_rpm(uint8_t esc_index, float& rpm) const
     return false;
 }
 
+// get the highest ESC RPM if available, returns true if there is valid data for at least one ESC
+bool AP_ESC_Telem::get_highest_motor_rpm(float& rpm) const
```

Comment:
Why highest and not average?

Reply:
It is possible to display both. Highest helps prevent prop failure by over-reving